openfecwebapp/routes.py

```diff
@@ -253,12 +254,12 @@ def legal_search(query, result_type):
 def advisory_opinions(query, offset):
     result_type = 'advisory_opinions'
     results = {}
-
     # Only hit the API if there's an actual query
     if query:
         results = api_caller.load_legal_search_results(query, result_type, offset=offset)
-
-    return views.render_legal_doc_search_results(results, query, result_type)
+        return views.render_legal_doc_search_results(results, query, result_type)
+    else: 
+        return views.render_legal_advisory_opinion_landing()
```

Comment:
I would put this in it's own route. I don't feel strongly about the endpoint names, though. Maybe the existing `advisory-opinions` and `regulations` search should move under `/legal/search/advisory-opinions` and `/legal/search/regulations`. which would free up `/legal/advisory-opinions/` for the landing page.

Eventually, I was envisioning something like `/legal/advisory-opintions/<AO number>` for the canonical AO pages, so doing something like the above fits with that.